### PR TITLE
Explore left-aligned website

### LINKED ISSUE #1761: Investigate left aligned text on the website
### Describe the change
Align all the text on the left. And maybe also think about alignment of all other things.

### Motivation
To make text easier to read through and navigate the website.

### Current implementation
image

### Suggested implementation
image

### PR DESCRIPTION
**Pull request by se-bastiaan**
Closes #1761 

### Summary
This changes the website design to mostly use left-aligned text, instead of centering everything.

### How to test
Steps to test the changes you made:
1. Check every single page.

### CODE REVIEW

File: website/merchandise/templates/merchandise/index.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 {% block body %}
     <section class="page-section">
         <div class="container">
-            <h1 class="text-center section-title">{% trans "Merchandise" %}</h1>
+            <h1 class="section-title">{% trans "Merchandise" %}</h1>
 
             <p class="text-center">{% trans "Below you can find all the merchandise Thalia has available. During every lunch break, the merchandise is for sale in the board room (M1.0.08). Outside those hours, please make an appointment or approach the board over drinks." %}</p>
```
Comment (pingiun) [inline website/merchandise/templates/merchandise/index.html:12]:
```suggestion
            <p>{% trans "Below you can find all the merchandise Thalia has available. During every lunch break, the merchandise is for sale in the board room (M1.0.08). Outside those hours, please make an appointment or approach the board over drinks." %}</p>
```

**Review by pingiun — COMMENTED** [review]
I suggest doing a `grep -ri 'text-centered'` in the codebase because there's still a lot that's centered.

I do really like the concept of left aligning text. For large blocks of text it's easier to read and I also like the look better. I do want to suggest making the text width smaller.

For example this piece of text contains 165 characters in width, while smaller line lengths [are advised](https://practicaltypography.com/line-length.html) for legibility. 

image

**Comment by pingiun** [comment]
> There are several other things that annoy me: the search bars were centred and I am not entirely sure how to place them.

I think keeping the search bar centered looks good.

For example on this member search page a centered search field does not look out of place:

image

**Review by pingiun — DISMISSED** [review]

**Review by ColonelPhantom — APPROVED** [review]
I only checked the code I own, but the changes there are trivial.

The left-alignment looks very nice!

**Comment by JobDoesburg** [comment]
image
I do feel that this pseudo-element in the divisors should be left-aligned too

**Comment by JobDoesburg** [comment]
image
The title here has some padding that should not be there (`#page-content #photos-index h1.section-title, #page-content #photos-album h1.section-title`)

**Comment by JobDoesburg** [comment]
image
What do we think about putting the main partner on the right here instead of in the center?

**Comment by JobDoesburg** [comment]
image
The `<ul>` on the right misses the list elements

**Comment by JobDoesburg** [comment]
image

**Review by Jorritboer — DISMISSED** [review]
I like it!

**Comment by se-bastiaan** [comment]
> image
> What do we think about putting the main partner on the right here instead of in the center?

Good idea, but maybe add that in another PR/issue? I have to rebase because of that page already 😅 



> image
> The list on the right misses the list elements

That's because of the `list-unstyled` class on the `ul`. Which is probably there because a list looked ugly on the centered text.

**Comment by JobDoesburg** [comment]
> That's because of the `list-unstyled` class on the `ul`. Which is probably there because a list looked ugly on the centered text.

Yeah I understand, with the left-aligned website we should add it back

**Review by wdoeland — APPROVED** [review]
Partners and vacancy pages look good!

**Comment by se-bastiaan** [comment]
> > That's because of the `list-unstyled` class on the `ul`. Which is probably there because a list looked ugly on the centered text.
> 
> Yeah I understand, with the left-aligned website we should add it back

Changed

**Review by JobDoesburg — APPROVED** [review]

**Review by LCKnol — APPROVED** [review]

**Pull request merged**